activity-metrics/metric-template.md

```diff
@@ -35,6 +35,9 @@ An example implementation, for example a SQL or Elasticsearch query.
 ## 6. Known Implementations
 Examples of where and how metric is used. (include links to dashboard or location where metric is visible or is talked about having been used).
 
-## 7. External References (Literature)
+## 7. Test Cases (Examples)
+Sample inputs (including contexts) and expected outputs for this metric. Implementers can test their implementations against these test cases. For quantitative metrics, this could include a static repository with known metric results. For qualitative metrics, this may be more difficult.

```

Comment:
s/For quantitative metrics, this could include a static repository with known metric results./For quantitative metrics, this could include a static repository with known metric results, or just inputs and output.